doc/conf.py

```diff
@@ -0,0 +1,83 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+import ast
+import re
+
+with open("../pytd/version.py", "rb") as f:
+    version_re = re.compile(r"__version__\s+=\s+(.*)")
+    VERSION = str(
+        ast.literal_eval(version_re.search(f.read().decode("utf-8")).group(1))
+    )
+
+
+# -- Project information -----------------------------------------------------
+
+project = "pytd"
+copyright = "2019, Arm Treasure Data"
+author = "Arm Treasure Data"
+
+# The full version, including alpha/beta/rc tags
+release = VERSION
+
+
+# -- General configuration ---------------------------------------------------
+
+# Add any Sphinx extension module names here, as strings. They can be
+# extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
+# ones.
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.autosummary",
+    "sphinx.ext.doctest",
+    "sphinx.ext.githubpages",
+    "numpydoc",
+    "sphinx.ext.viewcode",
```

Comment:
might be better to use `sphinx.ext.linkcode` instead https://www.sphinx-doc.org/en/master/usage/extensions/linkcode.html like scikit-learn or [Chainer](https://github.com/chainer/chainer/pull/2758/).

Reply:
e2b70aa986bc6a232539fd277410aa48b7866871